src/Microsoft.Extensions.Caching.Redis/RedisCacheServiceCollectionExtensions.cs

```diff
@@ -9,26 +9,30 @@
 namespace Microsoft.Extensions.DependencyInjection
 {
     /// <summary>
-    /// Extension methods for setting up Redis distributed cache related services in an
-    /// <see cref="IServiceCollection" />.
+    /// Extension methods for setting up Redis distributed cache related services in an <see cref="IServiceCollection" />.
     /// </summary>
-    public static class RedisCacheServicesExtensions
+    public static class RedisCacheServiceCollectionExtensions
     {
         /// <summary>
         /// Adds Redis distributed caching services to the specified <see cref="IServiceCollection" />.
         /// </summary>
         /// <param name="services">The <see cref="IServiceCollection" /> to add services to.</param>
-        /// <returns>A reference to this instance after the operation has completed.</returns> 
-        public static IServiceCollection AddRedisCache(this IServiceCollection services)
+        /// <param name="setupAction">An action delegate to configure the provided <see cref="RedisCacheOptions"/>.</param>
+        public static void AddRedisCache(this IServiceCollection services, Action<RedisCacheOptions> setupAction)
```

Comment:
@Tratcher we wanted to have a non-void return type for the cases where there was an especially interesting return type, e.g. MVC's or EF's framework-specific builder interface. Just chaining IServiceCollection isn't terribly interesting.